config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  # get "services", to: "services#index"
+  # get "services/new", to: "services#new"
+  # post "services", to: "services#create"
+  # get "services/:id", to: "services#show", as: :service
+  # get "services/:id/edit", to: "services#edit"
+  # patch "services/:id", to: "services#update"
+  # delete "services/:id", to: "services#destroy"
+  resources :users do
+    resources :appointments do
+      resources :services
```

Comment:
I think we should avoid double nesting these. I also think we shouldn't nest in to users. You can always use current_user